Library/Homebrew/utils/svn.rb

```diff
@@ -1,4 +1,8 @@
 module Utils
+  def self.clear_svn_version_cache
+    remove_instance_variable(:@svn) if instance_variable_defined?(:@svn)
```

Comment:
This is never needed, except in tests, so this shouldn‘t exist in the actual code.

Reply:
How about adding a method in one of the support modules for tests to clear any instance variable? As in a method whose argument is the instance variable to be cleared. @MikeMcQuaid  @reitermarkus  @mistydemeo

Reply:
@mansimarkaur If by support modules you mean in the `spec_helper` or something then: yeh, that could work. @reitermarkus?

Reply:
Also, there's probably some times this method should be called anyway such as e.g. after a new installation of `svn` (and same thing for `git`)

Reply:
Yes, I meant adding a method in the `spec_helper` but if we want to call this cache clear method after a new installation, then we don't need to add it in the `spec_helper`. Which is a better alternative? Also, `utils/git.rb` already has a `Utils.clear_git_available_cache` to remove `@git`, `@git_path` and `@git_version`.

Reply:
@mansimarkaur I think this is fine as-is, then, thanks.